src/managed_data/schema.py

```diff
@@ -0,0 +1,65 @@
+"""this module contains classes to generate data that
+matches the schema of real responses"""
+
+from dataclasses import dataclass, field
+from typing import Literal
+
+RowType = Literal["row", "database"]
+
+
+@dataclass
+class FieldItem:
+    """item in fields in DescribeRow"""
+
+    columnId: str = "_col:Q2mLSmW4LNGDXAXCUUa1j"
```

Comment:
Shouldn't the example/test values be separated out? This will make them the default values.

Reply:
in principle, yes. the only place this is being used is in the tests, so the values themselves don't matter -- what's being checked are the keys (and later the types too). 

i've chosen to bake the defaults in here so that the code to create fake data is very sparse -- you merely have to initialize these classes

Reply:
i could replace them with things that are more explictly placeholders, e.g.:

```
columnId: str = "_col:placeholder"
```

Reply:
I've usually just left the defaults empty if there are none. 

Programmatically generating the schema sounds great. If you think this is a v2 task, add it to the v2 epic.